### DISCUSSION TITLE
Disable format in an error message

### DISCUSSION THREAD

**Discussion by nanoqsh** [pest-parser/pest#662] [Q&A] [answered]
When I display an `Error` I got a message like this:
```
  --> 18:17
   |
18 |                 ]␊
   |                 ^---
   |
   = expected array, boolean, null, number, object, or string
```
Can I get only text message like this `expected array, boolean, null, number, object, or string` without any formatting?

**Comment by bobbbay** [comment, marked as answer]
Yes! Try displaying Error's public `variant` field, instead.

**Reply by nanoqsh** [reply]
Thanks!
